### PR TITLE
Hydra Sponsorship

### PR DESCRIPTION
**Pull request by felixoi**

### CODE REVIEW

**Comment by Katrix** [comment]
Any reason for removing those entries in the .gitignore?

**Comment by felixoi** [comment]
They are still there, they were doubled.

**Review by Katrix — APPROVED** [review]

**Review by Ewoutvans — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,3 +44,32 @@ development environment, most of the defaults will do except you must set `appli
 authentication to the Sponge forums. In addition, the SSL certification authority of https://forums.spongepowered.org is
 not typically recognized by the JVM so you will either have to manually add the cert to your JVM or set 
 `discourse.api.enabled` to `false` in the configuration file.
+
+### Using Hydra
+
+Hydra is a drop-in replacement of the regular Scala compiler.
+Its design goal is to take advantage of the many cores available in modern hardware to parallelize compilation of Scala sources.This gives us the possibility to achieve a much faster compile time.
```
Comment (mbax) [inline README.md]: Space between sentences.

File: README.md [hunk 2]
```diff
@@ -48,3 +44,32 @@ development environment, most of the defaults will do except you must set `appli
...
+Kindly [triplequote](https://triplequote.com/) provides us with some licenses.
+If you got one of them and want to use Hydra follow these steps:
```
Comment (mbax) [inline README.md]: Replace "If you got one of them" with "If you have a license"

**Comment by phase** [comment]
This is waiting on a logo for Hydra from Triplequote.

File: README.md [hunk 3]
```diff
@@ -41,6 +42,36 @@ This is required in order for Ore to run. Learn more about PostgreSQL [here](htt
...
+
+1. Create the file `project/hydra.sbt`
+2. Put in this content into the newly created file:
+   ```
+   credentials += Credentials("Artifactory Realm",
+       "repo.triplequote.com",
+       "<username>",
+       "<password>")
+   resolvers += Resolver.url("Triplequote Plugins Releases", url("https://repo.triplequote.com/artifactory/sbt-plugins-release/"))(Resolver.ivyStylePatterns)
+   addSbtPlugin("com.triplequote" % "sbt-hydra" % "1.0.2")
```
Comment (felixoi) [inline README.md]: Reminder for me to make the version a placeholder

**Pull request merged**